src/cloud-api-adaptor/podvm-mkosi/mkosi.presets/system/mkosi.conf.d/ubuntu-sftp.conf

```diff
@@ -0,0 +1,15 @@
+[Match]
+Distribution=ubuntu
+Profile=sftp
+PathExists=../../resources/authorized_keys
+
+[Content]
+Autologin=true
```

Comment:
Is `autologin=true` expected for sftp variant? I'd only expect it on debug variant, here I'd rely on the `sshd` service